core/Tracker/Response.php

```diff
@@ -100,15 +102,20 @@ public function outputResponse(Tracker $tracker)
 
     private function outputAccessControlHeaders()
     {
-        $requestMethod = isset($_SERVER['REQUEST_METHOD']) ? $_SERVER['REQUEST_METHOD'] : 'GET';
-
-        if ($requestMethod !== 'GET') {
+        if (!$this->isHttpGetRequest()) {
             $origin = isset($_SERVER['HTTP_ORIGIN']) ? $_SERVER['HTTP_ORIGIN'] : '*';
             Common::sendHeader('Access-Control-Allow-Origin: ' . $origin);
             Common::sendHeader('Access-Control-Allow-Credentials: true');
         }
     }
 
+    private function isHttpGetRequest()
+    {
+        $requestMethod = isset($_SERVER['REQUEST_METHOD']) ? $_SERVER['REQUEST_METHOD'] : 'GET';
+
+        return strtoupper($requestMethod) === 'GET';
+    }
+
```

Comment:
Maybe it would make sense to extract that somewhere global and to reuse it whereever the request method is checked like https://github.com/piwik/piwik/blob/master/core/Tracker.php#L244-L248 or https://github.com/piwik/piwik/blob/master/plugins/Annotations/Controller.php#L109 and so on